src/lib.rs

```diff
@@ -123,11 +123,13 @@ pub fn encode<T: AsRef<[u8]>>(data: T) -> String {
 // (4, 5)
 // (6, 7)
 // ...
+#[inline]
```

Comment:
I'm pretty sure this inline attribute (and the one below) are useless. The optimizer pretty much ignores the attribute and since these functions are private and thus only used in the same crate they are defined in, adding `#[inline]` doesn't make sense. 

Out of curiosity: why did you add it?

Reply:
Well, criterion told me, that there was a performance improvement, but I am not quite sure how reliable it is. (the next run it told me, that the performance decreased by 6%)